git_machete/annotation.py

```diff
@@ -9,15 +9,17 @@ def __init__(self, annotation: str):
         self.__push_text: str = ''
         self.__rebase_text: str = ''
         self.__slide_out_text: str = ''
+        self.__merge_text: str = ""
         self.rebase: bool = True
         self.push: bool = True
         self.slide_out: bool = True
+        self.merge: bool = False
 
-        def match_pattern(text: str) -> str:
-            return f'.*\\b{text}=no\\b.*'
+        def match_pattern(text: str, value: str = "no") -> str:
+            return f".*\\b{text}={value}\\b.*"
```

Comment:
TBH I'd just move the entire qualifier into the `text` &mdash; so that the calls look like `match_pattern("rebase=no")` or `match_pattern("update=merge")`.

Same for `def sub_pattern`